static/src/stylesheets/module/_adslot.scss

```diff
@@ -466,12 +466,12 @@
     }
 }
 .ad-slot--im {
-    box-sizing: border-box;
-    max-width: 50%;
     float: left;
+    margin: 5px $gs-gutter $gs-baseline 0;
```

Comment:
It's not a complete 1:1 similarity, as rich links get wider and have a negative left margin on desktop.